src/core/mesh/qgstopologicalmesh.h

```diff
@@ -190,9 +273,15 @@ class CORE_EXPORT QgsTopologicalMesh
     //! Checks the topology of the face and sets it counter clock wise if necessary
     static QgsMeshEditingError counterClockWiseFaces( QgsMeshFace &face, QgsMesh *mesh );
 
-    //! Reindexes faces and vertices, after this operation, topological can't be edited anymore
+    /**
+     * Reindexes faces and vertices, after this operation, the topological
+     * mesh can't be edited anymore and only the method mesh can be used to access to the raw mesh.
+     */
     void reindex();
 
+    //! method used for test and debug
```

Comment:
comment should describe what the function does, not who uses it... if the function is private and only for testing, maybe you can move it to private section and use `friend` class for test?